spring-boot-parent/spring-boot-custom-parent-example/src/main/java/com/baeldung/customparent/SpringBootStarterCustomParentApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.customparent;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SpringBootStarterCustomParentApplication {
```

Comment:
Will you please leave the word "example" out of the two directories? Technically, everything in this repo is an example.